CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # [Versions](https://mui.com/versions/)
 
+## 7.0.0-beta.2
+
+<!-- generated comparing v7.0.0-beta.1..master -->
+
+_Feb 27, 2025_
+
+A big thanks to the 2 contributors who made this release possible. Here are some highlights ✨:
```

Comment:
I suppose there are no noteworthy highlights for this release?

```suggestion
A big thanks to the 2 contributors who made this release possible.:
```

Reply:
Updated 😊